Hackney.Core.Tests/xunit.runner.json

```diff
@@ -0,0 +1,3 @@
+{
+  "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json"
```

Comment:
What is this used for?

Reply:
It's just a placeholder if you want to specify any particular config for the Xunit test runner